src/ft2font.cpp

```diff
@@ -515,16 +522,15 @@ FT2Font::FT2Font(FT_Open_Args &open_args, long hinting_factor_) : image(), face(
     } else if (error == FT_Err_Invalid_File_Format) {
         throw std::runtime_error("Can not load face.  Invalid file format.");
     } else if (error) {
-        throw std::runtime_error("Can not load face.");
+        throw_ft_error("Can not load face", error);
     }
 
     // set a default fontsize 12 pt at 72dpi
     hinting_factor = hinting_factor_;
 
-    error = FT_Set_Char_Size(face, 12 * 64, 0, 72 * (unsigned int)hinting_factor, 72);
-    if (error) {
+    if (int error = FT_Set_Char_Size(face, 12 * 64, 0, 72 * (unsigned int)hinting_factor, 72)) {
```

Comment:
Why do you need to re-declare `error`?

Reply:
I didn't even pay attention to the fact that the variable was declared above and could be reused, sure, changed.